### DISCUSSION TITLE
Feature request: Add marks and YouTube integration

### DISCUSSION THREAD

**Discussion by PabloDinella** [olive-editor/olive#1827] [Ideas]
Hello, first of all, thanks for the amazing work!

I would like to suggest two features: 

1. Ability to add marks and export them as .txt in the same format of youtube "chapters", like: 
```
00:00 Introduction
01:12 Hands on
06:00 Finishing
```
2. Maybe direct youtube upload.

I used to use those in Davinci Resolve, and find it very useful.

**Comment by DaniSeeh** [comment]
The first is being worked on (at least timeline labels are). No idea on exporting in that format 

The second would be much lower priority than a lot of other things. Perhaps a nice QoL thing for a few people, but personally I want color grading tools first. Matt is working on the main architecture and stability primarily right now.
